plugin/trino-prometheus/src/main/java/io/trino/plugin/prometheus/PrometheusClient.java

```diff
@@ -213,13 +213,19 @@ private static Interceptor basicAuth(String user, String password)
                 .build());
     }
 
-    private static Interceptor tokenAuth(String accessToken, String httpAuthHeaderName)
+    private static Interceptor tokenAuth(String accessToken, String httpAuthHeaderName, Map<String, String> additionalHeaders)
```

Comment:
Why not set additional HTTP headers in basic auth? Can we revert this `tokenAuth` method change and set those headers regardless of authentication?